src/main/java/core/basesyntax/service/impl/WriterServiceImpl.java

```diff
@@ -0,0 +1,18 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.WriterService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+public class WriterServiceImpl implements WriterService {
+
```

Comment:
remove the first empty line in all classes/interfaces/methods where it is present